src/ProjectNeon/Assets/Features/Effects/ConditionalEffect.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+using System;
+using UnityEngine;
+
+public abstract class ConditionalEffect : Effect
+{
+    protected Effect _effect;
```

Comment:
Don't store temporal params in class fields.

Reply:
@SilasReinagel We need to store the decorated effect! How are we supposed to propagate the effects without it?